message_sender/tasks.py

```diff
@@ -283,6 +288,8 @@ def run(self, message_id, **kwargs):
         else:
             # This is for retries based on async nacks from the transport.
             l.info("Message <%s> at max retries." % str(message_id))
+            message.to_addr = None
+            message.save()
```

Comment:
I think since this is in our core message loop, we want to make sure that we're only updating the `to_addr` field, to make sure that things are still fast:

https://docs.djangoproject.com/en/1.11/ref/models/instances/#specifying-which-fields-to-save